lib/core_ext/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -1,184 +1,186 @@
 module ActiveRecord
   module ConnectionAdapters # :nodoc:
-    module SchemaStatements # :nodoc:
```

Comment:
As a simplification, I'm patching PostgreSQL directly instead of the abstract adapter. Is there a reason for the prior configuration (considering this gem is specific to postgres)

Reply:
The idea I think was to deal with scenarios where you have multiple databases in one application that you have to deal with.